app/src/main/res/layout/no_internet.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Whole layout file can be changed to something like this (take a look at first lint warning): 

```
<?xml version="1.0" encoding="utf-8"?>
<TextView xmlns:android="http://schemas.android.com/apk/res/android"
    android:id="@+id/no_internet_view"
    android:layout_width="wrap_content"
    android:layout_height="wrap_content"
    android:layout_gravity="center"
    android:drawableTop="@drawable/ic_warning"
    android:fontFamily="sans-serif-regular"
    android:paddingBottom="120dp"
    android:text="@string/no_internet"
    android:textColor="@color/black_38"
    android:textSize="18sp" />
```